index.html

```diff
@@ -2030,6 +2030,111 @@ <h4>Presentations Using Derived Credentials</h4>
             </figcaption>
           </figure>
         </section>
+        <section>
+          <h4>Presentations Including Holder Claims</h4>
+          <p>
+A <a>holder</a> MAY use the <code>verifiableCredential</code> <a>property</a> in
+a <a>verifiable presentation</a> to include <a>verifiable credentials</a> from
+any <a>issuer</a>, including themselves. When the <a>issuer</a> of a
+<a>verifiable credential</a> is the <a>holder</a>, the <a>claims</a> in that
+<a>verifiable credential</a> are considered to be <em>self-asserted</em>.
+Such self-asserted claims can be secured by the same mechanism that secures
+the <a>verifiable presentation</a> in which they are included or by any
+mechanism usable for other <a>verifiable credentials</a>.
+          </p>
+          <p>
+The <a>subject(s)</a> of these self-asserted <a>claims</a> are not limited, so
+these <a>claims</a> can include statements about the <a>holder</a>, one of the
+other included <a>verifiable credentials</a>, or even the <a>verifiable
+presentation</a> in which the self-asserted <a>verifiable credential</a> is
+included. In each of these cases, the <code>CredentialSubject</code> would
+have an <code>id</code> <a>property</a> that matches the <code>id</code> of
+the <a>subject</a>, just as it is done in <a>verifiable credentials</a> that
+are not self-asserted.
```

Comment:
To ensure we're covering all subjects (not just top-level credential subjects):

```suggestion
included. In each of these cases, the <code>id</code> <a>property</a>
is used to identify the specific <a>subject</a> in the object where the
<a>claims</a> about it are made, just as it is done in
<a>verifiable credentials</a> that are not self-asserted.
```